### ISSUE TITLE
Nickname required for drop-off?

### ISSUE THREAD

**Issue by kylebrowning** [uber/rides-ios-sdk#1]
I couldn't seem to get the drop off location to pre-fill unless I used added a nickname, is this intentional?

```obj-c
[_requestButton setPickupLocationWithLatitude:pickupLat longitude:pickupLong nickname:nil address:nil];
```

**Comment by christinekim** [comment]
Yep, this is intended behavior. We do not do reverse-geocoding on the drop off location to display a name in the Uber app. In order to get the pin to display, you need to send along an address or nickname.

Thanks for pointing this out. I'll make sure our documentation is updated to clearly reflect this behavior.

**Comment by jbrophy17** [comment]
Fixed in [0.5.0](https://github.com/uber/rides-ios-sdk/releases/tag/v0.5.0)
